platform/macos/src/MGLMapView.mm

```diff
@@ -1633,13 +1633,14 @@ - (BOOL)gestureRecognizer:(NSGestureRecognizer *)gestureRecognizer shouldAttempt
 #pragma mark Keyboard events
 
 - (void)keyDown:(NSEvent *)event {
-    if (event.modifierFlags & NSNumericPadKeyMask) {
-        // This is the recommended way to handle arrow key presses, causing
-        // methods like -moveUp: and -moveToBeginningOfParagraph: to be called
-        // for various standard keybindings.
-        [self interpretKeyEvents:@[event]];
-    } else {
-        [super keyDown:event];
```

Comment:
I'm not quite clear how should I go about calling `[super keyDown:event]` in this method. Should I check if the event contains known keystrokes or call `super` in any case?